components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.server/src/test/java/org/wso2/carbon/identity/configuration/mgt/server/cors/util/TestUtils.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.configuration.mgt.server.cors.util;
+
+import org.apache.commons.dbcp.BasicDataSource;
+import org.apache.commons.lang.StringUtils;
+
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.spy;
+
+/**
+ * Utility class for CORSServiceTest
```

Comment:
Shall we add a `.` at the end of the comment? Also, shall we fix this in other respective files as well?